shared/src/test/scala/squants/thermal/TemperatureSpec.scala

```diff
@@ -288,7 +288,12 @@ class TemperatureSpec extends FlatSpec
     c.toString(Kelvin) should be("273.15°K")
     c.toString(Fahrenheit) should be("32.0°F")
     c.toString(Celsius) should be("0.0°C")
-    c.toString(Rankine) should be("491.66999999999996°R")
+
+    if (squants.Platform.name == "native") {
```

Comment:
With Scala Native, the Double toString ends up rounding to "491.67", so until that behavior is fixed I adjusted the tests.